### PR TITLE
feat(Graph2): Various improvements for ECL Watch integration

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage Please Review

File: packages/react/src/text.tsx [hunk 1]
```diff
@@ -2,26 +2,74 @@ import { Utility } from "@hpcc-js/common";
...
+                height={height}
+                fontFamily={fontFamily}
+                fill={fill}
+                y={yOffset + i * (height + 2)}
+            />
+        </g>;
+    });
+
+    return <>{TextLines}</>;
+};
```
Comment (jbrundage) [inline packages/react/src/text.tsx]: This seems like `Text` is trying to do too much. The ability to line wrap text isn't always needed. Seems like this functionality could be added by leaving `Text` alone and doing this:
```
export const WrappingText: React.FunctionComponent<IWrappingText> = ({
...
}) => {
    const TextLines = parts.map((p, i) => {
        return <g transform={`translate(0 ${yOffset + i * (height + 2)})`}>
            <Text
                text={p}
                height={height}
                fontFamily={fontFamily}
                fill={fill}
            />
        </g>;
    });
    return <g>{TextLines}</g>;
};
```

**Comment by GordonSmith** [comment]
@jbrundage plrease re-review

File: packages/react/src/text.tsx (hunk 1 above)
Comment (GordonSmith) [reply packages/react/src/text.tsx]: Is this just a naming thing?  I have "TextLine" + "Text" - you have "Text" + "WrappingText"?

Comment (GordonSmith) [reply packages/react/src/text.tsx]: I have removed "y" from Text and moved that logic into the <g> element as per your suggestion above.

**Pull request merged**